.github/workflows/bleedingedge.yml

```diff
@@ -0,0 +1,38 @@
+name: Nightly build
+
+on:
+  push:
+    branches: [ master ]
+  pull_request:
+    branches: [ master ]
+
+jobs:
+  test_nightly:
```

Comment:
I like the idea of doing this "nightly." One question--don't you need to use the [documented cron syntax](https://help.github.com/en/actions/reference/events-that-trigger-workflows#scheduled-events-schedule) with a `schedule` directive for this?

Reply:
I think I used a misleading name here. It's testing against the nightly build of Python3.9.
I had envisaged this PR as a CI test for that purpose, rather than a nightly build of SciPy. It's good to know that kind of thing is also possible.